app/views/users/index.html.erb

```diff
@@ -1,8 +1,15 @@
-<h1>User list</h1>
-<% @users.each do |user| %>
-  <div style="border: 2px solid black; margin: 5px">
-    <img src="<%= user.photo %>", style='height:100px;width:100;margin-bottom:20px;' >
-    <h3><%= user.name %></h3>
-    <p>Number of posts by <%= user.name %>: <%= user.posts_counter %></p>
+  <% @users.each do |user| %>
+  <div class="card mb-3">
+  <div class="row g-0">
+    <div class="col-md-4">
+      <img src=<%= user.photo %> class="img-fluid rounded-start" alt=<%= user.name %> >
+    </div>
+    <div class="col-md-8">
+      <div class="card-body">
+        <h5 class="card-title"><%= user.name %></h5>
+        <p class="card-text float-left"><small class="text-muted float-left">Number of Post: <%= user.posts_counter %></small></p>
+      </div>
+    </div>
```

Comment:
- [OPTIONAL] I noticed that these lines of code are the same as that of the partial `_user`. It would be nice to just render the partial instead of duplicating the code. 👍 😎